<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpPortDialog</class>
 <widget class="QDialog" name="IpPortDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: '#0d3d4c';
}</string>
  </property>
  <widget class="QPushButton" name="doneBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 5px solid '#101014';
	font: 700 17pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1.4px;
	color: '#e3e3e3';
}

QPushButton::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>201</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: '#101014';
	border-radius: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 700 30pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1px;
	color: '#e3e3e3';
}</string>
   </property>
   <property name="text">
    <string>Multi Chat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 5px solid '#101014';
	font: 700 17pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 1.4px;
	color: '#e3e3e3';
}

QPushButton::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 5px solid '#101014';
	font: 700 16pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="cursorPosition">
    <number>9</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 5px solid '#101014';
	font: 700 16pt &quot;Yu Gothic UI&quot;;
	letter-spacing: 2px;
	color: '#e3e3e3';
	background-color: '#0d3d4c';
}

QLineEdit::hover {
	background-color: '#0c2f3a';
}</string>
   </property>
   <property name="text">
    <string>4000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
